src/directives/drawfeature.js

```diff
@@ -0,0 +1,298 @@
+ *         ng-class="{active: dfCtrl.measureAzimut.active}"
+ *         ng-model="dfCtrl.measureAzimut.active"></a>
+ *       <a
+ *         href
+ *         translate
+ *         ngeo-btn
+ *         ngeo-drawrectangle
+ *         class="btn btn-default ngeo-drawfeature-rectangle"
+ *         ng-class="{active: dfCtrl.drawRectangle.active}"
+ *         ng-model="dfCtrl.drawRectangle.active"></a>
+ *       <a
+ *         href
+ *         translate
+ *         ngeo-btn
+ *         ngeo-drawtext
+ *         class="btn btn-default ngeo-drawfeature-text"
+ *         ng-class="{active: dfCtrl.drawText.active}"
+ *         ng-model="dfCtrl.drawText.active"></a>
+ *     </ngeo-drawfeature>
+ *
+ * @htmlAttribute {boolean} ngeo-drawfeature-active Whether the directive is
+ *     active or not.
+ * @htmlAttribute {ol.Map} ngeo-drawfeature-map The map.
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname ngeoDrawfeature
+ */
+ngeo.drawfeatureDirective = function() {
+  return {
+    controller: 'ngeoDrawfeatureController',
+    scope: true,
+    bindToController: {
+      'active': '=ngeoDrawfeatureActive',
+      'map': '=ngeoDrawfeatureMap'
+    },
+    controllerAs: 'dfCtrl'
+  };
+};
+
+ngeo.module.directive('ngeoDrawfeature', ngeo.drawfeatureDirective);
+
+
+/**
+ * @param {!angular.Scope} $scope Scope.
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$sce} $sce Angular sce service.
+ * @param {gettext} gettext Gettext service.
+ * @param {angularGettext.Catalog} gettextCatalog Gettext service.
+ * @param {ngeo.DecorateInteraction} ngeoDecorateInteraction Decorate
+ *     interaction service.
+ * @param {ngeo.FeatureHelper} ngeoFeatureHelper Ngeo feature helper service.
+ * @param {ol.Collection.<ol.Feature>} ngeoFeatures Collection of features.
+ * @constructor
+ * @ngInject
+ * @ngdoc controller
+ * @ngname ngeoDrawfeatureController
+ */
+ngeo.DrawfeatureController = function($scope, $compile, $sce, gettext,
+    gettextCatalog, ngeoDecorateInteraction, ngeoFeatureHelper, ngeoFeatures) {
+
+  /**
+   * @type {ol.Map}
+   * @export
+   */
+  this.map;
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.active;
+
+  if (this.active === undefined) {
+    this.active = false;
+  }
+
+  /**
+   * @type {angularGettext.Catalog}
+   * @private
+   */
+  this.gettextCatalog_ = gettextCatalog;
+
+  /**
+   * @type {ngeo.DecorateInteraction}
+   * @private
+   */
+  this.ngeoDecorateInteraction_ = ngeoDecorateInteraction;
+
+  /**
+   * @type {ngeo.FeatureHelper}
+   * @private
+   */
+  this.featureHelper_ = ngeoFeatureHelper;
+
+  /**
+   * @type {ol.Collection.<ol.Feature>}
+   * @private
+   */
+  this.features_ = ngeoFeatures;
+
+  /**
+   * @type {Array.<ol.interaction.Interaction>}
+   * @private
+   */
+  this.interactions_ = [];
+
+  /**
+   * @type {ol.interaction.Draw}
+   * @export
+   */
+  this.drawPoint;
```

Comment:
Mmm... I liked having those listed here.  Those 6 draw/measure directives can only be used within this main drawFeature directive, and those properties identify them correctly.  If one looks at this file only, one will understand the different interactions that can be bound from here.

I'm not against removing them.  I just find it useful.  Also, I haven't tested but this makes the properties correctly exported by closure, I think.  Otherwise, we would probably need to do `drawFeatureCtrl['drawPoint'] = drawPoint`.

Bottom line: I'm in favour of keeping them.  What do you think ?